src/components/notifications/Notifier.tsx

```diff
@@ -35,20 +35,18 @@ function Notifier() {
 
 	const notificiationStore = useNotificationsStore();
 
-	const prevResponseErrors = React.useRef<NotificationError[]>(notificiationStore.errors);
+	const prevResponseErrors = usePrevious(notificiationStore.errors);
 
 	useEffect(() => {
-		reaction(() => notificiationStore.errors, onNotificationsUpdate);
-	}, []);
+		reaction(() => notificiationStore.errors, onNotificationsUpdate, { fireImmediately: true });
+	}, [onNotificationsUpdate]);
```

Comment:
if there are errors this will be an infinite loop

delete this dependency, move function itself inside useEffect